activity-monitor.js

```diff
@@ -103,13 +118,35 @@
                 if (service.user.active && service.user.action <= inactive) {
                     service.user.active = false;
                     publish(EVENT_INACTIVE);
-                    disable();
+
+                    if(service.options.disableOnInactive){
+                        disable();
+                    }else{
+                        disableIntervals();//user inactive is known, lets stop checking, for now
+                        dynamicActivity = reactivate;//hot swap method that handles document event watching
+                    }
                 }
             }, service.options.monitor * MILLISECOND);
         }
 
+        /* function that lives in memory with the intention of being swapped out */
+        function dynamicActivity(){
+            regularActivityMonitor();
+        }
+
+        /* after user inactive, this method is hot swapped as the dynamicActivity method in-which the next user activity reactivates monitors */
+        function reactivate() {
+            enableIntervals();
+            dynamicActivity = regularActivityMonitor;
+        }
+
         /* invoked on every user action */
-        function activity() {
+        function activity(){
+            dynamicActivity()
```

Comment:
I need this with some quickness, please allow me to help you and here is a suggested CHANGELOG.md starter for you. You do not need to highlight each file change individually, people should use the github interface for file line changes. Hopefully this will copy/paste correctly:

```
# angular-activity-monitor - Change Log
All notable changes to this project will be documented here.

## [1.1.0] - 2016-08-23
### Added
- CHANGELOG.md
- disableOnInactive key to options. This modified event handling and required minor version bump
```